src/forms/CalendarComponent.jsx

```diff
@@ -32,8 +32,18 @@ class CalendarComponent extends React.Component {
 			onClose: this.onClose,
 			altInput: true,
 			allowInput: true,
-			altFormat: 'D M d, Y', // TODO localize
+			altFormat: 'D M d, Y',
 			defaultDate: this.props.value,
+			nextArrow: `<span class="svg svg--chevron-right">
```

Comment:
Is there a reason we can't just render the `<Icon>` component instead of manually adding the markup?

Reply:
these config options for `nextArrow` and `prevArrow` expect `html`
https://chmln.github.io/flatpickr/options/

so I don't think the component will work. I'm pretty sure I tried it a while back.